spinalcordtoolbox/scripts/sct_compute_flow.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+#
+# Compute flow from velocity encoding (VENC) sequence, based on the MRI phase image. More details in: https://mriquestions.com/what-is-venc.html
+#
+# Copyright (c) 2023 Polytechnique Montreal <www.neuro.polymtl.ca>
+# License: see the file LICENSE
+
+import sys
+from typing import Sequence
+
+from spinalcordtoolbox.image import Image, add_suffix
+from spinalcordtoolbox.qmri import flow
+from spinalcordtoolbox.utils.shell import Metavar, SCTArgumentParser, display_viewer_syntax
+from spinalcordtoolbox.utils.sys import init_sct, printv, set_loglevel
+
+
+def get_parser():
+    parser = SCTArgumentParser(
+        description="""
+        Compute flow from velocity encoding (VENC) sequence, based on the MRI phase image.
+        More details in: https://mriquestions.com/what-is-venc.html"""
+    )
+
+    mandatoryArguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatoryArguments.add_argument(
+        "-i",
+        required=True,
+        help="4D phase image. The 4th dimension should be the velocity encoding (VENC) in cm/s.",
+        metavar=Metavar.file,
+    )
+    mandatoryArguments.add_argument(
+        "-venc",
+        required=True,
+        help="Maximum velocity encoding (VENC) in cm/s.",
+        metavar=Metavar.float,
+    )
+
+    optional = parser.add_argument_group('\nOPTIONAL ARGUMENTS')
+    optional.add_argument(
+        '-o',
+        metavar=Metavar.file,
+        help='Output filename. Example: velocity.nii.gz')
+    optional.add_argument(
+        "-h",
+        "--help",
+        action="help",
+        help="Show this help message and exit")
+    optional.add_argument(
+        '-v',
+        metavar=Metavar.int,
+        type=int,
+        choices=[0, 1, 2],
+        default=1,
+        # Values [0, 1, 2] map to logging levels [WARNING, INFO, DEBUG], but are also used as "if verbose == #" in API
+        help="Verbosity. 0: Display only errors/warnings, 1: Errors/warnings + info messages, 2: Debug mode")
+
+    return parser
+
+
+def main(argv: Sequence[str]):
+    """Main function
+
+    Args:
+        argv (Sequence[str]): Command-line arguments
+    """
+    parser = get_parser()
+    arguments = parser.parse_args(argv)
+    venc = float(arguments.venc)
+    verbose = arguments.v
+    set_loglevel(verbose=verbose)
+
+    printv(f"Load data: {arguments.i}", verbose)
+    nii_phase = Image(arguments.i)
+
+    # Scale phase data between -pi and pi
+    printv('Scale phase data between -pi and pi', verbose)
+    data_phase_scaled = flow.scale_phase(nii_phase.data)
+
+    # Calculate velocity
+    printv('Calculate velocity', verbose)
+    velocity = flow.calculate_velocity(data_phase_scaled, venc)
```

Comment:
I'm a bit confused here. The current steps are:

```python
scale_phase()         # [-4096, 4094] -> [-π, π]
calculate_velocity()  # [-π, π] -> [-venc, venc]
```

In other words, `compute_flow()` is not actually called right now, and the only operations that happen are scaling operations.

Is this intentional? (The reason I ask is because I see two versions of the `compute_flow()` function, one of which is commented out, so I'm wondering if this is still a WIP PR.)

Reply:
JCA: The `compute_flow` functions are not called right now because people are not actually interested in the flow, they want the velocity.

My question is: If all this CLI script does is scale the data, is that necessary?

Reply:
JCA is currently using this function as-is, so it's OK to merge as-is without adding these new features.